apim-is-as-km/api-manager/provisioner/product_provisioner.sh

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+# set variables
+WSO2_SERVER=wso2am
+WSO2_SERVER_VERSION=2.1.0
+WSO2_SERVER_PACK=${WSO2_SERVER}-${WSO2_SERVER_VERSION}*.zip
+MYSQL_CONNECTOR=mysql-connector-java-5.1.*-bin.jar
+JDK_ARCHIVE=jdk-8u*-linux-x64.tar.gz
+WUM_ARCHIVE=wum-1.0-linux-x64.tar.gz
+WORKING_DIRECTORY=/home/vagrant
+JAVA_HOME=/opt/java/
+WUM_HOME=/usr/local
+WUM_PATH=PATH=$PATH:/usr/local/wum/bin
+DEFAULT_MOUNT=/vagrant
+CONFIGURATIONS=${DEFAULT_MOUNT}/api-manager/confs
+NODE_IP=$(/sbin/ifconfig eth1 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')
```

Comment:
The IP is pre-defined isn't it, so we can use it here?

Reply:
Yes. I removed them